src/components/SchemaTable/index.jsx

```diff
@@ -51,16 +51,24 @@ const useStyles = makeStyles(theme => ({
    *  on how many keywords the given schema or sub-schema defines)
    */
   line: {
+    display: 'flex',
+    alignItems: 'center',
     margin: 0,
     whiteSpace: 'nowrap',
-    minHeight: theme.spacing(3),
+    height: theme.spacing(3.5),
   },
   /**
    * Highlighting the type for the schema or sub-schema displayed
    * in the left panel of the schema table component.
    */
   code: {
     backgroundColor: theme.palette.grey[300],
+    padding: `0 ${theme.spacing(0.5)}px`,
+  },
+  tooltip: {
+    fontSize: `${theme.typography.subtitle2.fontSize}`,
+    lineHeight: 1,
```

Comment:
The `lineHeight` seems to have been the fix. Awesome work tracking this down! It was a tricky one.